tests/PySys/apt-plugin/apt-install/pysystest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+
+    <description>
+        <title>Validate apt plugin install</title>
+        <purpose><![CDATA[
+]]>
+        </purpose>
+    </description>
+    <classification>
+        <groups inherit="true">
+            <group></group>
+        </groups>
+        <modes inherit="true">
+        </modes>
+    </classification>
+    <data>
+        <class name="AptPluginTest" module="run"/>
```

Comment:
**If** you decide to change the class name to `AptPluginInstallTest`, you'll have to change it here as well.